### PR TITLE
Fix #615 - Expanded card bottom is right colour

### LINKED ISSUE #615: Make disclosure header grey when expanded
image

### PR DESCRIPTION
**Pull request by maxneuvians**
Fixes #615

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,26 +8,26 @@ https://cdsvacpoc.azurewebsites.net/
 
 ## Quickstart
 
-* `yarn install`
```
Comment (sastels) [inline README.md:11]: you don't like `*` apparently.

File: components/benefit_cards_b.js [hunk 2]
```diff
@@ -332,7 +337,7 @@ export class BenefitCardB extends Component {
               expanded={this.state.open}
               className={
                 this.state.open
-                  ? classes.ExpansionPanelOpen
```
Comment (sastels) [inline components/benefit_cards_b.js:335]: So can we 🔥 the `ExpansionPanelOpen` class in the style section?

**Review by sastels — APPROVED** [review]
minor comment, LGTM.
(could have been 2 PRs, would   have improved your GitHub stats 😛 )

**Review by smcmurtry — APPROVED** [review]
Looks good! 🦑

File: README.md (hunk 1 above)
Comment (maxneuvians) [reply README.md:11]: I think `prettier` did that!

File: components/benefit_cards_b.js (hunk 2 above)
Comment (maxneuvians) [reply components/benefit_cards_b.js:335]: Good call!

File: README.md (hunk 1 above)
Comment (sastels) [reply README.md:11]: huh - new prettier has different pretty-ness standards maybe?

**Pull request merged**